angular/core/components/group_form/group_form.haml

```diff
@@ -60,12 +60,12 @@
         .group-form__permissions.lmo-form-group
           %label{translate: "group_form.permissions"}
 
-          %label.lmo-form-labelled-input{ng-if: "group.privacyIsClosed() && !group.isSubgroupOfHiddenParent"}
+          %label.lmo-form-labelled-input{ng-if: "group.privacyIsClosed() && !group.isSubgroupOfSecretParent()"}
             %input.group-form__allow-public-threads{type: "checkbox", ng-model: "buh.allowPublicThreads", ng-change: "allowPublicThreadsClicked()"}
             %span{translate: "group_form.allow_public_threads"}
 
-          %label.lmo-form-labelled-input{ng-if: "group.isSubgroup() && !group.privacyIsSecret()"}
-            %input.group-form__visible-to-parent-members{type: "checkbox", ng-model: "group.parentMembersCanSeeDiscussions"}
+          %label.lmo-form-labelled-input{ng-if: "group.isSubgroup() && group.privacyIsClosed()"}
```

Comment:
This option only makes sense for closed groups.

parentMembersCanSee(Private)Discussions